src/devices/tuya.ts

```diff
@@ -3653,6 +3653,20 @@ const definitions: DefinitionWithExtend[] = [
             await reporting.bind(device.getEndpoint(1), coordinatorEndpoint, ['genOnOff']);
         },
     },
+    {
+        zigbeeModel: ['TS0001'],
+        model: 'X701A',
+        vendor: 'Lonsonho',
```

Comment:
Please modify `lonsonho.ts` instead of `tuya.ts`

Reply:
Ok, moved to `lonsonho.ts`.